src/containers/profile/src/constants/CaseStatusConstants.js

```diff
@@ -6,3 +6,12 @@ export const CLOSED :string = 'Closed';
 export const INTAKE :string = 'Intake';
 export const REFERRAL :string = 'Referral';
 export const RESOLUTION :string = 'Resolution';
+
+export const REASONS_FOR_CLOSED_CASE :string[] = [
+  'Successful completion',
+  'Respondent - Declined CRC',
+  'Respondent – Did not Respond/ Make Contact with CRC',
+  'Terminated- Rearrest',
+  'Terminated- Non-compliant',
+  'Terminated- At Respondent request',
```

Comment:
Minor issue: did you want to have a space before the `-` for all these strings? (for consistency).

Reply:
Oh shoot, sorry I merged this before seeing this comment. I copied the list exactly as they sent it to me! But you're right, I think it would look better with a space. I'll change in another PR.